static/app/views/replays/deadRageClick/selectorTable.tsx

```diff
@@ -0,0 +1,123 @@
+import {useCallback, useMemo} from 'react';
+import type {Location} from 'history';
+
+import renderSortableHeaderCell from 'sentry/components/feedback/table/renderSortableHeaderCell';
+import useQueryBasedColumnResize from 'sentry/components/feedback/table/useQueryBasedColumnResize';
+import useQueryBasedSorting from 'sentry/components/feedback/table/useQueryBasedSorting';
+import GridEditable, {GridColumnOrder} from 'sentry/components/gridEditable';
+import Link from 'sentry/components/links/link';
+import {Organization} from 'sentry/types';
+import useOrganization from 'sentry/utils/useOrganization';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+import {
+  DeadRageSelectorItem,
+  DeadRageSelectorQueryParams,
+} from 'sentry/views/replays/types';
+
+interface UrlState {
+  widths: string[];
+}
+
+interface Props {
+  clickCountColumn: {key: string; name: string};
+  clickType: 'count_dead_clicks' | 'count_rage_clicks';
```

Comment:
Is `clickType` the same as `clickCountColumn.key`?
it looks like clickType is in reference to a specific data field, and it's used for sorting. 

And the clickCountColumn is an additional column to add, but it's also set to be sortable. So i think clickType could go away and replaced with `clickCountColumn.key`